.devcontainer/images/.claude/commands/refine/render.md

```
@@ -157,6 +157,42 @@ description and no DONE clause), synthesis emits a single
+   && grep -qE '^## Testing Strategy' "$PLAN"; then
+  FORM=proof-triplet
+else
+  FORM=square-prompt   # v1.6 fallback, unchanged (GI4)
+fi
+```
+
+**proof-triplet** emits a copy-pasteable block where each numbered line is a
+**soldered triplet** (criterion + command + expected result), drawn from the
+plan's `[final-state]` acceptance lines:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)